### DISCUSSION TITLE
Remarks

### DISCUSSION THREAD

**Discussion by Versace69** [kieranholland/best-python-cheat-sheet#3] [Ideas]
Hi,

Just copying my reddit answer here.

Thanks for the cheat sheet.
I like the direct link to doc related for each section.

Some remarks I have noted about it:

- [x] 1. Python icon is to go back to the top, I found it by chance... Maybe it will be better to just write "Back to top" or change the icon to an arrow ? I was about to request it just before I found it...
- [x] 2. Full deep black for background with text above is very tiring to read. Looks like you are using Stackoverflow CSS. Look on Stackoverflow or Reddit, dark theme has more a greyish background than full deep black.
- [x] 3. Sections are well separated and visible in Light theme, that's not the case in Dark theme.
- [x] 4. Your banner at top should be used as a navigation bar as it stays always visible. Would be more useful in that case. Maybe to go back to the top of the visible section, navigate to previous and next section, going back to top ? Expand when mouse over to show direct link of summary ?

**Comment by kieranholland** [comment]
All suggestions have been incorporated - thanks

**Reply by Versace69** [reply]
Happy to see that.
I really like the dark theme color now. Github inspiration ahah :)
Usefull 3 navigation buttons also are a great add.
